includes/class-lipscore-widget-helper.php

```diff
@@ -17,7 +17,10 @@ public function product_attrs( $productData ) {
             'ls-price'          => $productData['price'],
             'ls-price-currency' => $productData['currency'],
             'ls-category'       => $productData['category'],
-            'ls-gtin'           => $productData['gtin']
+            'ls-gtin'           => $productData['gtin'],
+            'ls-product-url' => $productData['product_url'],
```

Comment:
I've confirmed with Therese that we don't need other `product_url` data and it's enough to use current `url` attribute.
So here it could be 
`'ls-product-url' => $productData['url']`
and then I think we can completely remove the new `product_url`from data builder